AMSoftware.Crm.PowerShell.Commands/Administration/GetSPDocLoc.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AMSoftware.Crm.PowerShell.Common;
+using AMSoftware.Crm.PowerShell.Common.ArgumentCompleters;
+using AMSoftware.Crm.PowerShell.Common.Repositories;
+using Microsoft.Crm.Sdk.Messages;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Query;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AMSoftware.Crm.PowerShell.Commands.Administration
+{
+    [Cmdlet(VerbsCommon.Get, "CrmSPDocLoc", HelpUri = HelpUrlConstants.GetRoleHelpUrl, SupportsPaging = true, DefaultParameterSetName = "")]
+    [OutputType(typeof(Entity))]
+    public class GetSPDocLoc :CrmCmdlet
+    {
+        [Parameter(Mandatory = true)]
```

Comment:
Include validation on the parameter if it is mandatory.
```suggestion
        [Parameter(Mandatory = true)]
        [ValidateNotNullOrEmpty]
```

Reply:
accepted